crates/sui-core/src/authority.rs

```diff
@@ -990,12 +990,33 @@ impl AuthorityState {
             return Err(anyhow!("dry-exec is only support on fullnodes"));
```

Comment:
nit, whilst we're here:

```suggestion
            return Err(anyhow!("dry-exec is only supported on fullnodes"));
```